sqlglot/dialects/snowflake.py

```diff
@@ -309,6 +315,7 @@ class Generator(generator.Generator):
             exp.DateStrToDate: datestrtodate_sql,
             exp.DataType: _datatype_sql,
             exp.DayOfWeek: rename_func("DAYOFWEEK"),
+            exp.Extract: _snowflake_generate_extract,
```

Comment:
can you just do exp.Extract: rename_func("DATE_PART")

Reply:
Ah nice, even cleaner